jsettlers.logic/src/jsettlers/logic/buildings/Building.java

```diff
@@ -81,12 +84,12 @@
 	private static final EPriority[] SUPPORTED_PRIORITIES_FOR_CONSTRUCTION = new EPriority[] { EPriority.LOW, EPriority.HIGH, EPriority.STOPPED };
 	private static final EPriority[] SUPPORTED_PRIORITIES_FOR_NON_WORKERS = new EPriority[0];
 
-	private static final ConcurrentLinkedQueue<Building> allBuildings = new ConcurrentLinkedQueue<Building>();
+	private static final ConcurrentLinkedQueue<Building> ALL_BUILDINGS = new ConcurrentLinkedQueue<Building>();
```

Comment:
But it does not define the term "constant".

Definition by wikipedia: "In computer programming, a constant is an identifier with an associated value which cannot be altered by the program during normal execution"

That queue can be altered in a sense that the new value is not equal to the old value.